packages/paste-core/components/popover/src/Popover.tsx

```diff
@@ -40,8 +40,9 @@ const Popover = React.forwardRef<HTMLDivElement, PopoverProps>(({children, ...pr
       {/* import Paste Theme Based Styles due to portal positioning. */}
       <StyledBase>
         <PopoverArrow {...(popover as any)} />
-        <Box padding="space50" paddingLeft="space70" paddingRight="space70">
+        <Box padding="space80" paddingBottom="space70" paddingTop="space70">
```

Comment:
For paddingBottom and Top, why is it that they're 70 instead of 80?

Reply:
@nkrantz I realize I hadn't read the thread abt this, but might be good to write the reason here for posterity?

Reply:
Ah yes - we decided to go with slightly smaller padding on top and bottom for visual design (thanks dsys-pd!)